include/picongpu/particles/Manipulate.hpp

```diff
@@ -59,47 +84,18 @@ namespace particles
         typename T_SpeciesType = bmpl::_1,
```

Comment:
I think `#include <boost/mpl/placeholders.hpp>` should be added because of this line.